event.go

```diff
@@ -11,13 +11,13 @@ import (
 type Severity int
 
 const (
-	TimeFormat             = "2006-01-02 15:04:05-0700 (MST)"
-	TraceSeverity Severity = iota
-	DebugSeverity
-	InfoSeverity
-	WarnSeverity
-	ErrorSeverity
-	CriticalSeverity
+	TimeFormat                = "2006-01-02 15:04:05-0700 (MST)"
+	TraceSeverity    Severity = 1
+	DebugSeverity    Severity = 2
+	InfoSeverity     Severity = 3
+	WarnSeverity     Severity = 4
+	ErrorSeverity    Severity = 5
+	CriticalSeverity Severity = 6
```

Comment:
The behaviour of `iota` is a little non intuitive so I think being explicit here is better, this is unrelated to the other changes.